static/app/views/replays/detail/layout/utils.tsx

```diff
@@ -0,0 +1,18 @@
+import theme from 'sentry/utils/theme';
+import {LayoutModes} from 'sentry/views/replays/detail/layout';
+
+export const getDefaultLayout = (): LayoutModes => {
+  const {innerWidth, innerHeight} = window;
+  const sidebarWidth = parseInt(theme.sidebar.expandedWidth, 10);
+  const windowsWidth = innerWidth - sidebarWidth;
+
+  if (innerHeight > 960 && windowsWidth < 1700) {
```

Comment:
> Can we do something that considers the height as well? I'm really curious to see how the proportion of the window can affect things.
> 
> 1024 seems reasonable on my laptop screen, but with different screen resolutions on external screens, ie: 2K, 4k I think it's not flexible enough to really change the output that people see.

Ok this has been a hard decision, but I believe this is a good spot - So imo I belive that if you have less than 1024px of width it is always more comfortable using the `topbar` layout.

But also, having more than 960px of height feels right to use the `topbar` as long as the width isn't at least 1700px - which I think in the case of having 1700px or more, the `sidebar` layout feels more comfortable overall.

Reply:
When I make the window take up the left 50% of my screen it chooses the sidebar layout, which seems wrong:
![Screen Shot 2022-08-04 at 8 13 17 AM (2)](https://user-images.githubusercontent.com/187460/182883499-47ab5960-eb40-4a41-b53c-9bfebe53dcd6.jpg)

Can we do a calculation that's based on the _proportion_ of the window size without any hardcoded values.
If the height is larger than we choose topbar.
If the width (minus the sidebar) is larger then we choose one of the sidebars.